environment/src/main/java/com/sequenceiq/environment/network/service/LoadBalancerEntitlementService.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.environment.network.service;
+
+import javax.inject.Inject;
+import javax.ws.rs.BadRequestException;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
+import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
+import com.sequenceiq.cloudbreak.common.type.CloudConstants;
+import com.sequenceiq.common.api.type.PublicEndpointAccessGateway;
+
+@Component
+public class LoadBalancerEntitlementService {
+
+    @Inject
+    private EntitlementService entitlementService;
+
+    public void valiateNetworkForEndpointGateway(String cloudPlatform, String environmentName,
```

Comment:
validate is misspelled :( sorry since it's used in a few places

Reply:
nooooooooo...